src/lib/components/cloudprogressbar/CloudProgressBar.component.js

```diff
@@ -0,0 +1,160 @@
+//@flow
+import React from 'react';
+import styled, { css } from "styled-components";
+import * as defaultTheme from "../../style/theme";
+
+export type CloudProgressBarProps = {
+  percentage: number,
+  borderSize: string,
+  color?: string,
+  progressColor?: string,
+  label?: string,
+  fontCOlor?: string,
+  fontSizePercentage?: string,
+  fontSizeLabel?: string
```

Comment:
In order to develop a more generic CloudProgressBar, I would propose to put all the stuff inside the cloud as children. So that the user can specify anything they want inside the cloud
`children: Node`